test/e2e/performanceprofile/functests/1_performance/irqbalance.go

```diff
@@ -0,0 +1,315 @@
+			Expect(postCreateBannedCPUs.IsEmpty()).To(BeFalse(), "banned CPUs %v not empty on node %q", postCreateBannedCPUs, node.Name)
+
+			By(fmt.Sprintf("getting a TuneD Pod running on node %s", node.Name))
+			pod, err := util.GetTunedForNode(cs, &node)
+			Expect(err).NotTo(HaveOccurred())
+
+			By(fmt.Sprintf("causing a restart of the tuned pod (deleting the pod) on %s", node.Name))
+			_, _, err = util.ExecAndLogCommand("oc", "delete", "pod", "--wait=true", "-n", pod.Namespace, pod.Name)
+			Expect(err).NotTo(HaveOccurred())
+
+			Eventually(func() error {
+				By(fmt.Sprintf("getting again a TuneD Pod running on node %s", node.Name))
+				pod, err = util.GetTunedForNode(cs, &node)
+				if err != nil {
+					return err
+				}
+
+				By(fmt.Sprintf("waiting for the TuneD daemon running on node %s", node.Name))
+				_, err = util.WaitForCmdInPod(5*time.Second, 5*time.Minute, pod, "test", "-e", "/run/tuned/tuned.pid")
+				return err
+			}).WithTimeout(5 * time.Minute).WithPolling(10 * time.Second).ShouldNot(HaveOccurred())
+
+			By(fmt.Sprintf("re-verifying worker node %q after TuneD restart", node.Name))
+			postRestartBannedCPUs, err := getIrqBalanceBannedCPUs(&node)
+			Expect(err).ToNot(HaveOccurred(), "failed to extract the banned CPUs from node %q", node.Name)
+			Expect(postRestartBannedCPUs).To(Equal(postCreateBannedCPUs), "banned CPUs changed post tuned restart on node %q", postRestartBannedCPUs, node.Name)
+		})
+
+		It("Should store empty cpu mask in the backup file", func() {
+			// crio stores the irqbalance CPU ban list in the backup file once, at startup, if the file doesn't exist.
+			// This _likely_ means the first time the provisioned node boots, and in this case is _likely_ the node
+			// has not any IRQ pinning, thus the saved CPU ban list is the empty list. But we don't control nor declare this state.
+			// It's all best effort.
+
+			nodeIdx := pickNodeIdx(workerRTNodes)
+			node := &workerRTNodes[nodeIdx]
+			By(fmt.Sprintf("verifying worker node %q", node.Name))
+
+			By(fmt.Sprintf("Checking the default IRQ affinity on node %q", node.Name))
+			smpAffinitySet, err := nodes.GetDefaultSmpAffinitySet(node)
+			Expect(err).ToNot(HaveOccurred(), "failed to get default smp affinity")
+
+			By(fmt.Sprintf("Checking the online CPU Set on node %q", node.Name))
+			onlineCPUsSet, err := nodes.GetOnlineCPUsSet(node)
+			Expect(err).ToNot(HaveOccurred(), "failed to get Online CPUs list")
+
+			// expect no irqbalance run in the system already, AKA start from pristine conditions.
+			// This is not an hard requirement, just the easier state to manage and check
+			Expect(smpAffinitySet.Equals(onlineCPUsSet)).To(BeTrue(), "found default_smp_affinity %v, expected %v - IRQBalance already run?", smpAffinitySet, onlineCPUsSet)
+
+			origBannedCPUsFile := "/etc/sysconfig/orig_irq_banned_cpus"
+			By(fmt.Sprintf("Checking content of %q on node %q", origBannedCPUsFile, node.Name))
+			expectFileEmpty(node, origBannedCPUsFile)
+		})
```

Comment:
The orig_irq_banned_cpus is private crio data and shouldn't be referenced from an NTO test case. This kind of testing should be done from the crio repo.

Reply:
In the context of this fix, we need to have some component resetting the irqbalance ban mask to zero. This is what this change is about and this is what functests want to cover. We don't have APIs or any way to check this besides checking the private data of crio, until the crio improvements land.

Reply:
> I'm not an owner so you don't need my approval.

This can easily change.  `OWNERS` file here is only really used for the bot to pick reviewers.  I believe anyone in the OpenShift org can LGTM.